modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -7170,6 +7170,7 @@ export-layout=Layout Export
 export-my-x-as-vcards=Export My {0} as vCards
 export-object-definition=Export Object Definition
 export-object-folder=Export Object Folder
+export-object-root-model=Export Object Root Model
```

Comment:
Since the beginning of this PR, you named: Root Model Object Definition.
Maybe we should follow the same order (export root model object definition)? Do you want to check if the designer?